requirements.txt

```diff
@@ -1,3 +1,4 @@
-django
+django==4.1.8
```

Comment:
This is failing to be found in the unit tests

Reply:
I saw that. I will check for a correct django version